unlock-protocol-com/blog/grants-program-updates.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Unlock Announces Grants Program Upgrade
+subTitle: Improvements go live to streamline processes and accelerate growth for the Unlock ecosystem
+description: Improvements go live to streamline processes and accelerate growth for the Unlock ecosystem.
+author: Angela Steffens
+date: November, 1 2022
```

Comment:
@wonderwomancode date is in incorrect format (the comma is in the wrong place) so post does not render on the list of posts at ./blog/

Reply:
Fixed it